web/src/features/exchanges/ExchangeTooltip.tsx

```diff
@@ -17,10 +17,11 @@ export default function ExchangeTooltip({
   exchangeData,
   isMobile,
 }: ExchangeTooltipProperties): ReactElement {
-  const { key, netFlow, co2intensity } = exchangeData;
+  const { key, f, ci } = exchangeData;
```

Comment:
I'm sorry for pointing this out quite late, but I think we should try to do the remapping as early as possible instead of using the shortened version all the way down.

In this exchange we could make the remapping in `useExchangeArrowsData` so it's passed down :)

Reply:
Will take care of it later today 🙂

Reply:
Thanks, I know it's quite late to bring this up 😅